tests/client_connectivity/test_bridge_mode.py

```diff
@@ -0,0 +1,302 @@
+import pytest
+import sys
+
+for folder in 'py-json', 'py-scripts':
```

Comment:
Why do we need to mess with the Python path here?

Reply:
because lanforge scripts is not a module, hence we cant install it as a module and we rely on it being on a specific path

Reply:
@greearb please consider making your python libraries consumable as a module.